### PR TITLE
Implement /iam route

### PR DESCRIPTION
**Pull request by vastus**

### CODE REVIEW

**Comment by turansadri** [comment]
This code is shit

**Comment by sirtawast** [comment]
Yeah use perl

File: app.rb [hunk 1]
```diff
@@ -4,3 +4,7 @@
   "hey wadapp, fool?"
 end
 
+get "/iam" do
```
Comment (vastus) [inline app.rb]: wtf? why not pass the mood as a param?

File: app.rb [hunk 2]
```diff
@@ -4,3 +4,7 @@
...
+  "good"
```
Comment (turansadri) [inline app.rb]: I like the "good" part

**Comment by vastus** [comment]
> ego hits

File: app.rb [hunk 3]
```diff
@@ -4,3 +4,7 @@
   "hey wadapp, fool?"
 end
```
Comment (sirtawast) [inline app.rb:5]: I'm glad it ended prematurely. Like me.

File: app.rb [hunk 4]
```diff
@@ -4,3 +4,7 @@
   "hey wadapp, fool?"
```
Comment (liding1990) [inline app.rb:4]: wadatfapp

**Pull request merged**